global.json

```diff
@@ -1,12 +1,7 @@
 {
-  // This empty file has no effect and is here to support Arcade-powered dotnet-build.
-  // The Arcade build process requires a global.json at the root of the repo and normally
-  // uses it to restore the necessary SDK and tools to build the repo.
-  // However, NuGet explitcly wants to use the ambient SDK (normally latest), not any particular
-  // version.  We also don't want to introduce MS.DN.Arcade.SDK into the non-dotnet-build build.
-  // eng/dotnet-build/global.json has the entry for the Arcade version we need for
-  // Arcade-powered dotnet-build.
   "sdk": {
+    "version": "8.0.100",
```

Comment:
Wouldn't this choose 8.0.100 if it was available in addition to other versions?

I think in general we've preferred not using it. We have the .vsconfig to help handle the installation of required components and we target net8.0 in our tests, so you won't be able to build without the 8.0.100 and later SDKs.

Reply:
Updated it to latestMajor, thanks Andy. Yes, by using `latestMajor` the host enforces a required minimum version of the .NET SDK while at the same time keeping the existing "ambient" behavior that is preferred in this repository.

Using this over any .vsconfig file or so has the advantage that a developer coming into this repository and using `dotnet build`, will immediately get feedback if the SDK is lower than the minimum version. In general, this is a strong contract that allows to specific a minimum supported SDK version.